### PR TITLE
fix: getting locked out when running script

### PR DESCRIPTION
**Pull request by jeevithakannan2**
## Type of Change
- [x] Bug fix

## Description
- Prevents getting locked out when running scripts ( Might happen when using tty fonts, alacritty, kitty )
- Min TUI prompt will open once the floating window is closed.

## Testing
https://github.com/user-attachments/assets/03843582-b71e-4ff4-ad65-16110514b2e1

## Checklist
- [x] My code adheres to the coding and style guidelines of the project.
- [x] I have performed a self-review of my own code.
- [x] My changes generate no errors/warnings/merge conflicts.

### CODE REVIEW

File: tui/src/state.rs [hunk 1]
```diff
@@ -189,7 +189,8 @@ impl AppState {
         let terminal_size = frame.area();
 
         if !self.size_bypass
-            && (terminal_size.height < MIN_HEIGHT || terminal_size.width < MIN_WIDTH)
+            && !matches!(self.focus, Focus::FloatingWindow(_))
```
Comment (adamperkowski) [inline tui/src/state.rs:192]:
```suggestion
            && self.focus != Focus::FloatingWindow(_)
```

Comment (jeevithakannan2) [reply tui/src/state.rs:192]: This will not work

**Comment by jeevithakannan2** [comment]
Closing in favor of #933

**Comment by jeevithakannan2** [comment]
Closing in favor of https://github.com/ChrisTitusTech/linutil/pull/933

**Pull request closed without merging**